src/theme/foundations/colors.ts

```diff
@@ -21,6 +21,145 @@ import { hexToRGB } from "../../utils/utils";
  */
 
 /** Reusable variables: */
+// Primitives
+export const primitives = {
```

Comment:
I really like the names of the primitives, like `scienceBlue`. Too bad we won't use them elsewhere.

Reply:
Yes, I did have some fun finding appropriate names for the primitives. But alas, the colors used in designs should be the semantic colors.